MdePkg/Include/Protocol/PciIo.h

```diff
@@ -20,6 +20,8 @@
 
 typedef struct _EFI_PCI_IO_PROTOCOL EFI_PCI_IO_PROTOCOL;
 
+typedef struct _EFI_PCI_IO_DOE EFI_PCI_IO_DOE;

```

Comment:
MdePkg is for industry standard only, including UEFI/ACPI, etc.

Since EFI_PCI_IO_DOE is NOT in UEFI spec, it should be in MdeModulePkg, not MdePkg.

Reply:
Thanks! I have moved it to `MdeModulePkg`